.gitignore

```diff
@@ -21,3 +21,6 @@
 terraform/state.tf
 
 local/
+
+# OSX
+*.DS_Store
```

Comment:
This is fine here, but it's also a good practice to add this to your global `.gitignore`

Reply:
I've been using git for 5+ years and TIL there's a global `.gitignore`... LOL, thanks for the tip.